wagtail-{{ cookiecutter.project_name_kebab }}/.github/workflows/test.yml

```diff
@@ -8,24 +8,50 @@ on:
 
   pull_request:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
```

Comment:
Some comment as https://github.com/wagtail/cookiecutter-wagtail-package/pull/29#discussion_r1038042000 here. This is now the default

Reply:
Leaving it there on the "explicit > implicit"